subliminal/providers/legendastv.py

```diff
@@ -242,7 +242,7 @@ def search_titles(self, title):
                     if match:
                         title['season'] = int(match.group('season'))
                     else:
-                        logger.warning('No season detected for title %d', title_id)
+                        logger.warning('No season detected for title %d (%s)', title_id, title['title'])
```

Comment:
I'm not comfortable logging text from the provider without processing it. It can cause some encoding issues. Can't you use the title_id to retrieve it anyway?